DropDown/src/DropDown.swift

```diff
@@ -861,20 +842,20 @@ extension DropDown {
 	}
 
 	/// (Pre)selects a row at a certain index.
-	public func selectRowAtIndex(_ index: Index?) {
+	public func selectRow(atIndex index: Index?) {
```

Comment:
Wouldn't `selectRow(at index: Index?)` be nicer? Like the Swift 3 API for `UITableView` which have [`selectRow(at:animated:scrollPosition:)`](https://developer.apple.com/reference/uikit/uitableview/1614875-selectrow)